10bis/src/inject/inject.js

```diff
@@ -22,24 +22,23 @@ function initialIjection() {
 	var today = new Date();
 	var dayOfMonth = today.getDate();
 	var dayOfWeek = today.getDay();
-	if (dayOfMonth > 24) {
-		dayOfMonth -= daysOfMonth(today);
-	}
-
-	var daysLeft = 25 - dayOfMonth - ~~((25 - dayOfMonth + (dayOfWeek + 1) % 7) / 7) - ~~((25 - dayOfMonth + dayOfWeek) / 7);
+	var year = today.getFullYear()
+	var month = today.getMonth() + 1
+	var daysInMonth = parseInt(/[\d\.]+/.exec(document.querySelector(".reportGeneralDataFieldValueTd").innerText).input.slice(7,9))
```

Comment:
What if it's not at `slice(7, 9)`, e.g. "5/5- 4/6"? Better to use a regex...
Also, consider renaming `daysInMonth` to something more descriptive e.g. `lastDay`.